infra/terraform/environments/dev/main.tf

```diff
@@ -59,6 +59,13 @@ module "user_nfs_softnas" {
     dns_zone_domain = "${module.cluster_dns.dns_zone_domain}"
 }
 
+module "data_backup" {
+    source = "../../modules/data_backup"
+
+    env = "${var.env}"
+    k8s_worker_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/nodes.${var.env}.${data.terraform_remote_state.base.xyz_root_domain}"
```

Comment:
Would be nice to have interpolation in the `.tfvars` or intermediate variables...but currently it's not possible ([TF #4084](https://github.com/hashicorp/terraform/issues/4084) and [TF #10059](https://github.com/hashicorp/terraform/issues/10059)).